selection_sort/python/selection_sort.py

```diff
@@ -0,0 +1,34 @@
+def selection_sort(arr): # Function that performs the SelectionSort
+    """
+    - The SelectionSort function takes in an array as argument
+    - Sorts the elements in array by making use of inner and outer loops
+    - Returns a sorted array as output
+    :papram arr: An array for sorting
+    """
+    n = len(arr) # Length of arr is stored for further use
```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `all.pyjava`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmptcqwn69q/selection_sort/python/selection_sort.py
+++ b/tmp/tmptcqwn69q/selection_sort/python/selection_sort.py
@@ -12,7 +12,7 @@
         for j in range(i+1, n):
             """
             Traverse the array from i+1 to n-i as inner loop,
-            i.e. i is the counter for outer loop and j for 
+            i.e. i is the counter for outer loop and j for
             the inner loop
             """
             if arr[j] < arr[low]: # Set low to current j, if it's value is smaller
```